edge-hub/docker/linux/amd64/Dockerfile

```diff
@@ -3,7 +3,11 @@ FROM azureiotedge/azureiotedge-runtime-base:1.2-linux-amd64 as builder
 
 FROM mcr.microsoft.com/dotnet/core/aspnet:${base_tag}
 
-ARG EXE_DIR=.	
+ADD ./watchdog/target/x86_64-unknown-linux-musl/release/watchdog /usr/local/bin/watchdog
+ADD ./mqttd/target/x86_64-unknown-linux-musl/release/mqttd /usr/local/bin/mqttd
+ADD ./mqttd/production.json /tmp/mqtt/config/production.json
```

Comment:
I created /tmp/mqttd because the tmp directory is where the edgehub user has permission to write. The broker needs to write its state to this directory. Because I already had an mqttd folder, I chose to put the configuration here. Perhaps there is a better place?

Reply:
It's more up to the people who are working on the broker.  If you all are OK with it.

Reply:
I have changed this to read the config from /app